tests/twitter.test.ts

```diff
@@ -58,7 +58,7 @@ describe("Twitter", () => {
     expect(res).not.toBeUndefined();
   });
 
-  test("Fail to post Tweet", async () => {
+  test("Expect Tweet post failure on network error", async () => {
```

Comment:
Because the error shows in the log maybe it makes sense to just test for success case?

I'm sure I won't be the only confused reviewer in the future.